.tool-versions

```diff
@@ -0,0 +1 @@
+nodejs 16.14.0
```

Comment:
What is this file for?

Reply:
It is a file used by [asdf](https://github.com/asdf-vm/asdf) to define versions of tools, node in this case. It is like nvm but not specific for node.

I forgot to exclude it when I made the commit, I will update my commit and remove it later today.

Reply:
I updated my commit to remove it.